src/app/multi-line-chart/multi-line-chart.component.html

```diff
@@ -0,0 +1 @@
+<h2>{{ title || 'No title' }}</h2>
```

Comment:
Can we do a little better here and at the very least, funnel in the "Data TYpe" of the graph? (Cases, Deaths, Recovered)

Reply:
Yeah, I like that! I or you can add in a future PR :D